Automattic/wp-calypso
# Stats: Avoid showing low scores in January

#### Proposed Changes

* show the previous year's annual stats in January

#### Testing Instructions



* open a live branch
* navigate to the Insights page and verify that 2022 is visible in the annual highlight component
* verify that you can still see 2023 when using the arrows on the right-hand side
* go to another blog and check that it loads 2022 instead of 2023

#### Pre-merge Checklist





Related to #72106
